main.go

```diff
@@ -181,7 +189,7 @@ func startWebhookServer(c conf.Configuration, disableWebhookConfigInstaller bool
 	}
 }
 
-func runAudit(c conf.Configuration, auditPath string, setExitCode bool, outputFile string, outputURL string, outputFormat string) {
+func runAudit(c conf.Configuration, auditPath string, outputFile string, outputURL string, outputFormat string) validator.AuditData {
```

Comment:
nit pick - feels like the validator.AuditData class should hold the `marshal` functionality of the auditdata itself, instead of in `runAudit()`.

Reply:
Not sure I agree here - marshaling is a pretty trivial task, and one I'd generally leave up to the user. Could be convinced though.

Reply:
In my mind it felt like a part of domain of the audit data. I just thought it would be nice to use methods like `func (ad AuditData) toJSON() []bytes` and `func (ad AuditData) toYAML() []bytes`.